installer/datafiles/ruby.data

```diff
@@ -3,38 +3,14 @@ RUBY_DEST:                '/opt/microsoft/omsagent/ruby'
 
 %Files
```

Comment:
@jeffaco 
The data file that I generated has the changes, that is correct, but I have not run the script with the SOURCE dir pointing to intermediate ruby paths at all. That is what I think is broken in the version of genRubyInstaller .